.gitignore

```diff
@@ -4,10 +4,11 @@
 .ipynb_checkpoints
 .ipynb_checkpoints/
 **/__pycache__
+*.egg-info
 
 tasks_data/*
 !tasks_data/.gitkeep
 
 
 **/params.local.sh
-*/_account_private_registry.sh
\ No newline at end of file
+*/_account_private_registry.sh
```

Comment:
Hm, this is something I don't remember adding.

Reply:
That's just the end of line character that was added, we have some files without the trailing end of line chars because of the different editor settings.

Reply:
Ah, correct :)